sw/host/opentitanlib/src/transport/ti50emulator/emu.rs

```diff
@@ -82,6 +89,45 @@ impl EmulatorProcess {
         return EMULATOR_INVALID_ID;
     }
 
+    pub fn get_configurations(&self) -> Result<EmulatorConfig> {
+        let args_list = vec![
+            OsString::from("--path"),
+            self.runtime_directory.clone().into_os_string(),
+            OsString::from("--get_configs"),
+        ];
+
+        let exec: PathBuf = match self.current_args.get("exec") {
```

Comment:
Is `PathBuf` required here?  I think the return value of `Path::join` is PathBuf.

Reply:
https://doc.rust-lang.org/stable/src/std/path.rs.html#2444-2446
Yes - It will not work with Path.